docs/FAQ/how-to-auto-deploy-bold-reports-multiple-services-in-docker-compose.md

```diff
@@ -0,0 +1,300 @@
+# How to auto deploy multiple services Bold Reports via docker-compose.
+This section allows you to deploy [Bold Reports](https://www.boldreports.com/) in docker-compose without manual activation of licensing and configuring startup from the browser.
+
+## Steps for Bold Reports multiple services auto deployment using docker-compose.
+1. Create an empty project directory.
+   You can name the directory something easy for you to remember. This directory is the context for your application image. This project directory should contains a `docker-compose.yml` file which is complete in itself for a good starter BoldReports project.
+
+2. Download the configuration files [here](https://github.com/boldreports/bold-reports-docker/tree/master/deploy/multiple-container). This directory includes docker-compose YML file and configuration file for Nginx.
+   > **Tip:** You can use either a `.yml` or `.yaml` extension for this file. They both works well.
+
+3. Change into your project directory. For example, if you named your directory my_boldreports
+   > $  cd my_boldreports/
+4. Create a docker-compose.yml file that starts your `BoldReports` and a separate `PostgreSQL` instance with volume mounts for data persistence:
+
+   ```sh
+
+     version: '3.5'
+
+     services:
+       id-web:
+         container_name: id_web_container
+         image: gcr.io/boldreports-294612/bold-identity:6.15.11

```

Comment:
Replace our last release image tag in the corresponding places.

Reply:
Replaced the last release image tags.